regenerate.sh

```diff
@@ -29,7 +29,7 @@ mkdir -p ${GOBIN}
 echo "remove existing generated files"
 # grpc_testing_not_regenerate/*.pb.go is not re-generated,
 # see grpc_testing_not_regenerate/README.md for details.
```

Comment:
Let's update this comment to talk about the new pb.go's as well

Reply:
sure, that make sense